### ISSUE TITLE
LinkedIn Live Comments don't have profile photo

### ISSUE THREAD

**Issue by wondergit113** [steveseguin/social_stream#320]
I have run a few livestreams and Social Stream brings in LinkedIn comments but not profile pictures. How can I help test/improve this?

**Comment by steveseguin** [comment]
Are there avatar photos for linkedin commenters in the linkedin chat itself?  If not, I might not be able to easily get them.

If there are, it might be a bug or perhaps linkedin changed things.  If you have a live stream going, just poke me with the link in the next short while, and I'll test /update./

**Comment by wondergit113** [comment]
Yes there are. I can setup a livestream to debug and will come back to you. Won't be today though. Is there a timeframe that normally works well for you?

**Comment by steveseguin** [comment]
I'm going to bed soon, as reference, though I am available anytime this week, assuming I'm awake.

**Comment by wondergit113** [comment]
I'm using 0.2.17 app on macOS. There will be an event at 4pm GMT today at this URL:
https://www.linkedin.com/events/7295095616773246976/ 

If you're around and can help debug, would be much appreciated!

**Comment by steveseguin** [comment]
Thank you. I should be available I think

**Comment by steveseguin** [comment]
![Image](https://github.com/user-attachments/assets/a8076cfc-8819-4d7c-aa5f-9bc93cfb0e9e)

Fixed. pushed to beta.

https://github.com/steveseguin/social_stream/commit/27a6a0a7045d0ed5eb95c52cd37bdcbb5202dde0
